streaming/java/streaming-runtime/src/test/java/io/ray/streaming/runtime/demo/WordCountTest.java

```diff
@@ -18,22 +18,33 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 public class WordCountTest extends BaseUnitTest implements Serializable {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(WordCountTest.class);
+  private static final Logger LOG = LoggerFactory.getLogger(WordCountTest.class);
 
   // TODO(zhenxuanpan): this test only works in single-process mode, because we put
   //   results in this in-memory map.
   static Map<String, Integer> wordCount = new ConcurrentHashMap<>();
 
+  private Object rayAsyncContext;
+
+  @BeforeClass
+  public void setUp() {
+    super.setUp();
+    Ray.init();
+    rayAsyncContext = Ray.getAsyncContext();
+  }
+
   @Test
   public void testWordCount() {
-    Ray.shutdown();
+    Ray.setAsyncContext(rayAsyncContext);
+
     StreamingContext streamingContext = StreamingContext.buildContext();
     Map<String, String> config = new HashMap<>();
-    config.put(Config.CHANNEL_TYPE, Config.MEMORY_CHANNEL);
+    config.put(Config.CHANNEL_TYPE, "MEMORY_CHANNEL");
```

Comment:
```suggestion
    config.put(Config.CHANNEL_TYPE, Config.MEMORY_CHANNEL.nameI());
```

Reply:
We need a better way to use Config. (in another PR) @chunyang-wen